lib/ansible/modules/cloud/vmware/vmware_dvswitch_nioc.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+
+# Copyright: (c) 2018, VMware, Inc.
+# SPDX-License-Identifier: GPL-3.0-or-later
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: vmware_dvswitch_nioc
+short_description: Manage distributed vSwitch Network IO Control.
+description:
+    - Manages distributed vSwitch Network IO Control.
+version_added: 2.8
+author:
+    - Joseph Andreatta (@vmwjoseph) <joseph () vmware.com>
+
+notes:
+    - Tested on vSphere 6.7
+requirements:
+    - "python >= 2.6"
+    - PyVmomi
+options:
+    switch_name:
+        description:
+            - The name of the switch to create or remove.
```

Comment:
We are not dealing with Standard switch, right ? Also, we are not creating or removing switch.

Reply:
Correct, this functionality is only for distributed switches. I've fixed the description.